lib/src/peppermint_functionalities/nft/repo/nft_repo.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:get/get.dart';
+import 'package:peppermint_sdk/peppermint_sdk.dart';
+import 'package:peppermint_sdk/src/api/api_list_response.dart';
+
+class NftRepo {
+  final GetConnect _walletClient;
+  final ErrorHandlers _errorHandler;
+  final String _baseUrl;
```

Comment:
no need to define baseUrl to repo.
should define baseUrl in GetConnect().
The same way as WalletConnect